airflow/cli/commands/db_command.py

```diff
@@ -47,8 +53,36 @@ def resetdb(args):
 def upgradedb(args):
     """Upgrades the metadata database"""
     print("DB: " + repr(settings.engine.url))
-    db.upgradedb(version_range=args.range, revision_range=args.revision_range)
-    print("Upgrades done")
+    if args.revision and args.version:
+        raise SystemExit("Cannot supply both `revision` and `version`.")
```

Comment:
```suggestion
        raise SystemExit("Cannot supply both `--revision` and `--version`.")
```

To be consistent with the others?